doc/manual/integers-and-floating-point-numbers.rst

```diff
@@ -700,7 +700,7 @@ imply multiplication:
     julia> (x-1)(x+1)
     ERROR: MethodError: `call` has no method matching call(::Int64, ::Int64)
     Closest candidates are:
-      Union(!Matched::Any...)
+      Union{!Matched::Any...}
```

Comment:
i think this one just happens to be a bit strange, and the parens are actually fairly correct (since it is specifically matching `call(::Type{Union}, args...) = Union{args...}` in essentials.jl:9). I think the bigger confusion there comes from the printing of `call` in prefix notation, but without the indication that the function itself didn't actually match.